src/main/java/com/duckblade/osrs/toa/features/scabaras/overlay/AdditionPuzzleSolver.java

```diff
@@ -75,31 +75,31 @@ public class AdditionPuzzleSolver implements PluginLifecycleComponent
 
 	private static final Map<Integer, Set<Integer>> OPTIMAL_SOLUTIONS = ImmutableMap.<Integer, Set<Integer>>builder()
 		.put(20, ImmutableSet.of(5, 11, 17))
-		.put(21, ImmutableSet.of(15, 16, 17, 18, 19))
-		.put(22, ImmutableSet.of(20, 21, 22, 23, 24))
-		.put(23, ImmutableSet.of(0, 6, 12, 18, 24))
+		.put(21, ImmutableSet.of(10, 11, 17))
+		.put(22, ImmutableSet.of(10, 11, 12, 18, 24))
+		.put(23, ImmutableSet.of(5, 6, 7, 8, 14))
 		.put(24, ImmutableSet.of(5, 11, 17, 23))
 		.put(25, ImmutableSet.of(10, 11, 12, 13))
-		.put(26, ImmutableSet.of(19, 20, 21, 22, 23, 24))
-		.put(27, ImmutableSet.of(10, 11, 12, 13, 18))
-		.put(28, ImmutableSet.of(4, 5, 6, 7, 8, 9))
+		.put(26, ImmutableSet.of(10, 11, 12, 13, 9))
+		.put(27, ImmutableSet.of(5, 6, 7, 8, 4))
+		.put(28, ImmutableSet.of(0, 1, 7, 13, 19))
 		.put(29, ImmutableSet.of(10, 11, 12, 13, 19))
 		.put(30, ImmutableSet.of(10, 11, 12, 13, 14))
-		.put(31, ImmutableSet.of(0, 5, 10, 15, 20))
-		.put(32, ImmutableSet.of(9, 14, 19, 20, 21, 22, 23, 24))
-		.put(33, ImmutableSet.of(0, 5, 10, 15, 16, 20))
+		.put(31, ImmutableSet.of(0, 6, 12, 13, 14))
+		.put(32, ImmutableSet.of(10, 6, 2, 3, 4))
+		.put(33, ImmutableSet.of(4, 5, 6, 7, 8, 9, 14))
 		.put(34, ImmutableSet.of(10, 11, 12, 13, 14, 19))
-		.put(35, ImmutableSet.of(0, 1, 6, 11, 16, 21))
+		.put(35, ImmutableSet.of(10, 11, 12, 13, 14, 9, 19))
 		.put(36, ImmutableSet.of(0, 1, 2, 3, 4, 9, 14))
 		.put(37, ImmutableSet.of(10, 11, 12, 13, 14, 19, 24))
-		.put(38, ImmutableSet.of(2, 8, 10, 11, 12, 13, 14))
-		.put(39, ImmutableSet.of(0, 1, 2, 3, 4, 9, 13))
-		.put(40, ImmutableSet.of(0, 5, 10, 15, 20, 21))
-		.put(41, ImmutableSet.of(0, 1, 2, 5, 10, 15, 20))
-		.put(42, ImmutableSet.of(0, 6, 12, 18, 20, 21, 22, 23, 24))
-		.put(43, ImmutableSet.of(0, 1, 2, 3, 5, 10, 15, 20))
-		.put(44, ImmutableSet.of(1, 2, 3, 4, 6, 11, 16, 21))
-		.put(45, ImmutableSet.of(0, 5, 10, 15, 16, 17, 18, 19))
+		.put(38, ImmutableSet.of(0, 5, 6, 12, 18, 24))
```

Comment:
I believe this one is missing 10. Can you confirm?

![image](https://user-images.githubusercontent.com/1868974/212814030-25ed67f4-cea3-41b8-9e11-6429105ebdb3.png)